security/pkg/stsservice/tokenmanager/google/tokenexchangeplugin.go

```diff
@@ -287,10 +288,10 @@ func (p *Plugin) sendRequestWithRetry(req *http.Request) (resp *http.Response, e
 }
 
 type accessTokenRequest struct {
-	Name      string        `json:"name"` // nolint: structcheck, unused
-	Delegates []string      `json:"delegates"`
-	Scope     []string      `json:"scope"`
-	LifeTime  time.Duration `json:"lifetime"` // nolint: structcheck, unused
+	Name      string            `json:"name"` // nolint: structcheck, unused
+	Delegates []string          `json:"delegates"`
+	Scope     []string          `json:"scope"`
+	LifeTime  duration.Duration `json:"lifetime"` // nolint: structcheck, unused
```

Comment:
I highly recommend you do *not* use the protobuf struct in a struct that is marshalled with `json.Marshal`. Its just asking for trouble. I don't think protobuf actually has any guarantee of the output of json.marshal on a proto object, since you are not supposed to do it.

Reply:
yep, lint also detected sync.Mutex inside Duration.